toolkit/committee-selection/query/src/lib.rs

```diff
@@ -163,7 +163,9 @@ where
 		let first_epoch = {
 			let second_block = (self.client)
 				.hash(1.into())
-				.map_err(err_debug)?
+				.map_err(|err| {
+					format!("Node is not in archive mode, not able to fetch first block: {err:?}")
```

Comment:
Node is not in archive mode therefore not able to fetch first block:
(just a suggestion, no fret)